.github/workflows/binary-applications.yml

```diff
@@ -251,9 +261,7 @@ jobs:
             - name: Set up GCP
               # Secrets not available in PR so don't use GCP.
               if: github.event_name != 'pull_request'
-              uses: google-github-actions/setup-gcloud@master
-              with:
-                  service_account_key: ${{ secrets.GOOGLE_SERVICE_ACC_KEY }}
+              uses: google-github-actions/setup-gcloud@v0

```

Comment:
Do we not need to authenticate in this build job?

Reply:
Actually, looking into the [sampledata build job on your fork](https://github.com/emlys/invest/runs/4720464510?check_suite_focus=true), I see a bunch of errors sprinkled throughout:

```
ServiceException: 401 Anonymous caller does not have storage.objects.create access to the Google Cloud Storage object.
```

So maybe we do need to authenticate?

Reply:
Oh, looks like I just missed that one!